components/Notification.tsx

```diff
@@ -75,11 +74,18 @@ export const Notification = (props: NotificationProps) => {
   return (
     <StyledNotification {...styleProps}>
       {isClosable && (
-        <button className="delete" onClick={handleCloseClick}>
-          <Icon isSmall>
-            <Icon.Title>Close Notification</Icon.Title>
-            <Icon.Close />
-          </Icon>
```

Comment:
It didn't like this, eh?

Reply:
Nope.  `npm run ds:build` fails, so the build wasn't building, which we need to catch moving forward.  But first things first.